CONTRIBUTION.md

```
@@ -0,0 +1,429 @@
+uv run pre-commit install
+
+# Run hooks manually on all files
+uv run pre-commit run --all-files
+
+# Skip hooks for a specific commit (use sparingly)
+git commit --no-verify -m "Your commit message"
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)